shells/dev/webpack.config.js

```diff
@@ -48,6 +39,15 @@ module.exports = {
             }
           }
         ]
+      },
+      {
+        test: /\.(png|jpg|gif|svg|eot|ttf|woff|woff2)$/,
+        use: [
+          {
+            loader: 'url-loader',
+            options: { limit: 0 }
+          }
+        ]
```

Comment:
This teaches the css-loader about images. It makes it so that `background-image: url("../../assets/icon.svg")` creates a dependency tracked by Webpack. Webpack can resolve it to a public path, run additional transformation (like image optimization), and give it a unique hash.